src/components/qas/questions/qaMultiSelect/index.jsx

```diff
@@ -0,0 +1,197 @@
+import { checkIds } from '../../../../lib/utilities.js';
+import ConditionalWrapper from '../../../ConditionalWrapper';
+import Card from '../../../site/card';
+import styles from './qaMultiSelect.module.scss';
+import CheckBox from '../../../site/icons/CheckBox';
+import CheckBoxOutlineBlank from '../../../site/icons/CheckBoxOutlineBlank';
+
+class QAMultiSelect extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    const { answer } = props;
+    const { data } = answer || {};
+
+    this.state = {
+      hasFocus: false,
+      answerable: false,
+      selectedOptions: data || [],
+      selectedValue: '',
+    };
+  }
+
+  componentDidUpdate() {
+    const { answerable } = this.state;
+    const { question, activeId, ids } = this.props;
+    const { id } = question;
+    const active = ids ? checkIds(ids, activeId) : activeId === id;
+
+    this.checkAnswerable(answerable, active);
+  }
+
+  checkAnswerable(answerable, active) {
+    if (!answerable && active) {
+      this.setState(prevState => ({
+        ...prevState,
+        answerable: true,
+      }));
+    }
+  }
+
+  handleChange = e => {
+    const {
+      question,
+      handleAnswerSelect,
+      focusCallback,
+      saveHandler,
+    } = this.props;
+    if (e) {
+      this.setState(
+        prevState => ({
+          ...prevState,
+          hasFocus: true,
+        }),
+        () => {
+          if (focusCallback) {
+            const { hasFocus } = this.state;
+            focusCallback(hasFocus);
+          }
+        }
+      );
+    }
+    if (!e) {
+      this.setState(
+        prevState => ({
+          ...prevState,
+          hasFocus: false,
+        }),
+        () => {
+          if (focusCallback) {
+            const { hasFocus } = this.state;
+            focusCallback(hasFocus);
+          }
+          const { selectedOptions } = this.state;
+          handleAnswerSelect(question.id, selectedOptions, e.type);
+          saveHandler();
+        }
+      );
+    }
+  };
+
+  handleStoredOptions = e => {
+    const { checked, value } = e.target;
+    if ((checked || value) !== undefined) {
+      const { selectedOptions } = this.state;
+      if (!checked) {
+        const index = selectedOptions.indexOf(value);
+        selectedOptions.splice(index, 1);
+      } else {
+        selectedOptions.push(value);
+      }
+      this.setState(prevState => ({
+        ...prevState,
+        selectedOptions,
+        selectedValue: selectedOptions.join(', '),
+      }));
+    }
+  };
+
+  getMultiSelectOptions = options => {
+    const { selectedOptions } = this.state;
+    return options.map((option, i) => {
+      return (
+        <SelectionControl
+          id={`qa-multi-select--${option.value}`}
+          checkedCheckboxIcon={<CheckBox />}
+          uncheckedCheckboxIcon={<CheckBoxOutlineBlank />}
+          // eslint-disable-next-line react/no-array-index-key
+          key={`${option.label}-${i}--option`}
```

Comment:
I think just take out the index from this key value and then you don't need to disable this rule.  It would I think never ever be the case that the label to two select options will be the same because then why would you have both options.

Reply:
Got it. Yeah, my editor looks like a crime scene when I take out the key, but as long as the app doesn't crash, haha